src/sentry/static/sentry/app/views/requiredAdminActions/setCallsigns.jsx

```diff
@@ -92,9 +91,10 @@ const SetCallsignsAction = createReactClass({
 
   onSetShortName(projectId, event) {
     this.setState({
-      slugs: update(this.state.slugs, {
-        [projectId]: {$set: event.target.value.toUpperCase().trim()},
-      }),
+      slugs: {
+        ...this.state.slugs,
+        [projectId]: event.target.value.toUpperCase().trim(),
```

Comment:
why are there brackets here? is it a dynamically resolved key?

Reply:
yes!